src/index.ts

```diff
@@ -182,15 +188,20 @@ const plugin: PluginImpl<Options> = (options = {}) => {
       const source = ts.createSourceFile(chunk.fileName, code, ts.ScriptTarget.Latest, true);
       const fixer = new NamespaceFixer(source);
 
-      code = Array.from(typeReferences, ref => `/// <reference types="${ref}" />`).join("\n");
-      if (code) {
-        code += "\n";
-      }
+      code = writeBlock(Array.from(typeReferences, ref => `/// <reference types="${ref}" />`));
+      code += writeBlock(Array.from(moduleDeclarations, moduleDeclaration => moduleDeclaration));
```

Comment:
```suggestion
      code += writeBlock(Array.from(moduleDeclarations));
```
I don’t think you need an identity transform here.